packages/core/src/sass/app/_common/components/platform-switcher.scss

```diff
@@ -5,6 +5,10 @@
     transition: background-color 0.3s;
     cursor: pointer;
 
+    &--dark-mode {
+        color: #fff;
+    }
```

Comment:
This code is wrong, we need to handle dark mode automatically using our theme.scss file!
please look into a better solution.

Reply:
The icons texts are transparent in the PR, but they do show when it's serving (as shown in screenshot), is that fine?

![Screenshot 2023-03-06 at 12 18 59 PM](https://user-images.githubusercontent.com/108507236/223018723-c4fbd5be-aad4-41ad-bfaa-8fe3f866c4d4.png)

![Screenshot 2023-03-06 at 12 18 26 PM](https://user-images.githubusercontent.com/108507236/223018761-b94a26bd-6e0e-45b0-b476-d6afb8dd0718.png)

I made a PR [here](https://github.com/hirad-rewok/deriv-app/pull/14)

Reply:
Should be okay here, this is probably because github cannot understand `var`